modules/about-workbench-images.adoc

```diff
@@ -102,11 +102,11 @@ The *CUDA - RStudio Server* workbench image contains NVIDIA CUDA technology. CUD
 endif::[]
 
 | ROCm
-| Use the ROCm notebook image to run AI and machine learning workloads on AMD GPUs in {productname-short}. It includes ROCm libraries and tools optimized for high-performance GPU acceleration, supporting custom AI workflows and data processing tasks. Use this image integrating additional frameworks or dependencies tailored to your specific AI development needs.
+| Use the ROCm workbench image to run AI and machine learning workloads on AMD GPUs in {productname-short}. It includes ROCm libraries and tools optimized for high-performance GPU acceleration, supporting custom AI workflows and data processing tasks. Use this image integrating additional frameworks or dependencies tailored to your specific AI development needs.
 
 | ROCm-PyTorch
-| Use the ROCm-PyTorch notebook image to optimize PyTorch workloads on AMD GPUs in {productname-short}. It includes ROCm-accelerated PyTorch libraries, enabling efficient deep learning training, inference, and experimentation. This image is designed for data scientists working with PyTorch-based workflows, offering integration with GPU scheduling.
+| Use the ROCm-PyTorch workbench image to optimize PyTorch workloads on AMD GPUs in {productname-short}. It includes ROCm-accelerated PyTorch libraries, enabling efficient deep learning training, inference, and experimentation. This image is designed for data scientists working with PyTorch-based workflows, offering integration with GPU scheduling.
```

Comment:
@coderabbitai explain how an average customer will understand the word _optimize_ on line 108

Reply:
I'd personally say "execute PyTorch workloads"; without special gpu support, the code cannot be executed on a gpu